.vscode/settings.json

```diff
@@ -0,0 +1,21 @@
+{
+  "workbench.colorCustomizations": {
+    "activityBar.activeBackground": "#65c89b",
+    "activityBar.activeBorder": "#945bc4",
+    "activityBar.background": "#65c89b",
+    "activityBar.foreground": "#15202b",
+    "activityBar.inactiveForeground": "#15202b99",
+    "activityBarBadge.background": "#945bc4",
+    "activityBarBadge.foreground": "#e7e7e7",
+    "statusBar.background": "#42b883",
+    "statusBar.border": "#42b883",
+    "statusBar.foreground": "#15202b",
+    "statusBarItem.hoverBackground": "#359268",
+    "titleBar.activeBackground": "#42b883",
+    "titleBar.activeForeground": "#15202b",
+    "titleBar.border": "#42b883",
+    "titleBar.inactiveBackground": "#42b88399",
+    "titleBar.inactiveForeground": "#15202b99"
+  },
+  "peacock.color": "#42b883"
+}
```

Comment:
.vscode ?

Reply:
arf, oui c'est vscode. des fenêtres de couleurs différentes quand j'ai plusieurs projets ouvert. J'ignorais que ces settings passait avec la PR

Reply:
Will have to add it to `.gitignore` ! Nothing to do with current project, and you could override all your team's local settings :scream: 

(As a sidenote, I use Peacock too, it's quite nice uh ? ^^)

Reply:
Put .vscode in .gitingore has done nothing. I will check later how to hide this kind of settings.  At this time i removed the folder .vscode.